app/code/Magento/Review/Model/ResourceModel/Rating.php

```diff
@@ -425,9 +433,11 @@ public function getReviewSummary($object, $onlyForCurrentStore = true)
 
         $data = $connection->fetchAll($select, [':review_id' => $object->getReviewId()]);
 
+        $currentStore = ($this->_state->getAreaCode() == "adminhtml") ? false : $this->_storeManager->getStore()->getId();
```

Comment:
I think we should use an existing constant for providing area name.
Also, please use NULL instead of false, we shouldn't mix types of variable. Store id is number or null type, but not false

Reply:
Hello @VladimirZaets , I'll check and update it asap.